instrumentation-core/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/TracerProperties.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Loads opentelemetry.trace.tracer.name and opentelemetry.trace.tracer.loggingExporterIsEnabled
+ * from application.properties
+ *
+ * <p>Sets the default values if the configurations do not exist
+ */
+@ConfigurationProperties(prefix = "opentelemetry.trace.tracer")
+public final class TracerProperties {
+
+  private String name = "otel-spring-tracer";
+  private boolean loggingExporterIsEnabled = true;
+
+  public String getName() {
+    return name;
+  }
+
+  public void setName(String tracerName) {
+    this.name = tracerName;
+  }
+
+  public boolean isLoggingExporterIsEnabled() {
```

Comment:
Do you think it's possible (or even a good idea) to recognize our standard system properties in the spring config instead of redefining? e.g., 

```
ota:
  exporter: logging
```

https://github.com/open-telemetry/opentelemetry-java-instrumentation#logging-exporter

Reply:
Currently we don't support the simple logging exporter and I don't think we need to. All it does is print spans to console and I am only using it to debug my changes. My goal is to use our standard system properties to support a few exporters out of the box. When I add that feature to this project I will either remove this configuration.

Reply:
Here's a branch which contains a rough draft for exporter support (it currently has LoggingExporter and JaegerExporter). Instead of setting system properties this approach uses the properties set in the resources/application.properties file. 

The proposed key is:
```
opentelemetry:
  exporter: 
     type:  logging (or jaeger or otlp or zipkin)
```

Reply:
What do you think?